src/opnsense/mvc/app/models/OPNsense/Routes/Route/Migrations/M0_0_1.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace OPNsense\Routes\Route\Migrations;
+use OPNsense\Base\BaseModelMigration;
+class M0_0_1 extends BaseModelMigration
+{
+    public function run($model)
+    {
+       $model.save();
```

Comment:
@fichtner I am also not sure. Do you have an idea how to populate the gateway list. It comes from a php function in `gwlb` which returns an array of associative arrays